src/entities/QuarterBudget/model.ts

```diff
@@ -195,4 +209,25 @@ export default class QuarterBudgetModel extends Model<QuarterBudgetAttributes> {
       categories: categoriesResults,
     }
   }
+
+  static parseBudgets(results: BudgetQueryResult[]): Budget[] {
+    const budgetsByQuarter = new Map<string, BudgetQueryResult[]>()
```

Comment:
Is `string` `BudgetQueryResult['id']`?

Reply:
Yes! But I'll change it to `BudgetQueryResult['id']`